.github/workflows/ci.yml

```diff
@@ -428,29 +429,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
 
-      - name: Setup Buf
-        uses: bufbuild/buf-setup-action@76ddbd1bcb9da6da11cb7c41bd97e47f81c39a39 # v1.37.0
+      - name: Lint and Push Protocol Buffers
+        uses: bufbuild/buf-action@v1
         with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.BUF_TOKEN }}
+          pr_comment: false
```

Comment:
But switched to one action that does all the things (lint, breaking changes, push, etc).